modules/serverless-kafka-source-kn.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_sources/serverless-kafka-source.adoc
+[id="serverless-kafka-source-kn_{context}"]
+= Creating a Kafka event source using the `kn` CLI
+
+This section describes how to create a KafkaSource using the `kn` command.
+
+.Prerequisites
+
+* The {ServerlessOperatorName}, `KnativeEventing`, and the `KnativeKafka` custom resource definitions are installed on your cluster.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have access to the Kafka cluster that produces the Kafka messages you want to import.
+
+.Procedure
+
+. To verify that the Kafka source is working, create a Knative service that dumps incoming messages into the service's logs:
```

Comment:
```suggestion
. To verify that the Kafka source is working, create a Knative service that dumps incoming messages into the service logs:
```

Reply:
If we're including a service in the example, we probably need to mention Knative Serving as a prereq above